src/Constraint.jl

```diff
@@ -40,10 +40,18 @@ end
 
 AbstractTrees.children(c::Constraint) = (c.child,)
 
-# A fallback. Define a new method if `MOI.Utilities.distance_to_set`
-# is not defined.
+# A fallback. Define a new method if `MOI.Utilities.distance_to_set` is not
+# defined.
 function is_feasible(x, set, tol)
-    return MOI.Utilities.distance_to_set(x, set) <= tol
+    try
+      return MOI.Utilities.distance_to_set(x, set) <= tol
+    catch
```

Comment:
I’ve found blanket catches can make debugging hard, but I think we only need to catch MethodErrors?
```suggestion
    catch e
    e Isa MethodError() || rethrow()
```

Reply:
The other option is just to opt-in to some sets